docs/developer-docs/latest/development/plugins-development.md

```diff
@@ -61,10 +61,16 @@ Strapi provides a [command line interface (CLI)](/developer-docs/latest/develope
 </code-group>
 
 7. (*TypeScript-specific*) Run `npm install` or `yarn install` in the newly-created plugin directory.
-8. Run `yarn build` or `npm run build` to build the plugin.
+8. (*TypeScript-specific*) Run `yarn build` or `npm run build` in the plugin directory. This step transpiles the JavaScript files in a `dist` directory unique to the plugin.
```

Comment:
This one should be, "This transpiles the typescript files and outputs the javascript to a dist directory unique to the plugin".

Reply:
OK, fixed